modules/imgproc/test/test_mine.cpp

```diff
@@ -0,0 +1,106 @@
+//
+// Created by amir on 06.12.20.
+//
+
+#include "test_precomp.hpp"
+
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+
+namespace opencv_test { namespace {
+
+    void print_extra(Rect &rect) {
+        float big_coord = 3.f * MAX( rect.width, rect.height );
+        float rx = (float)rect.x;
+        float ry = (float)rect.y;
+
+        auto topLeft = Point2f( rx, ry );
+        auto bottomRight = Point2f( rx + rect.width, ry + rect.height );
+
+        printf("x = [%.2f, %.2f, %.2f, %.2f, %.2f];\n", topLeft.x, topLeft.x, bottomRight.x, bottomRight.x, topLeft.x);
+        printf("y = [%.2f, %.2f, %.2f, %.2f, %.2f];\n", topLeft.y, bottomRight.y, bottomRight.y, topLeft.y, topLeft.y);
+        printf("plot(x, y, '-c');\n");
+        printf("hold on;\n");
+
+        Point2f ppA( rx + big_coord, ry );
+        Point2f ppB( rx, ry + big_coord );
+        Point2f ppC( rx - big_coord, ry - big_coord );
+
+        printf("plot(%.2f, %.2f, 'o-k');\n", ppA.x, ppA.y);
+        printf("hold on;\n");
+        printf("plot(%.2f, %.2f, 'o-k');\n", ppB.x, ppB.y);
+        printf("hold on;\n");
+        printf("plot(%.2f, %.2f, 'o-k');\n", ppC.x, ppC.y);
+        printf("hold on;\n");
+    }
+
+    void print_voronoi(Subdiv2D &subdiv) {
+        std::vector<std::vector<Point2f>> facetList;
+        std::vector<Point2f> facetCenters;
+        subdiv.getVoronoiFacetList({}, facetList, facetCenters);
+        for (size_t i = 0; i < facetList.size(); ++i) {
+            printf("x = [");
+            for (size_t j = 0; j < facetList[i].size(); ++j) {
+                printf("%.2f ", facetList[i][j].x);
+            }
+            printf("%.2f];\n", facetList[i][0].x);
+            printf("y = [");
+            for (size_t j = 0; j < facetList[i].size(); ++j) {
+                printf("%.2f ", facetList[i][j].y);
+            }
+            printf("%.2f];\n", facetList[i][0].y);
+            printf("plot(x, y, '-k');\n");
+            printf("hold on;\n");
+            printf("plot(%.2f, %.2f, 'o-k');\n", facetCenters[i].x, facetCenters[i].y);
+            printf("hold on;\n");
+        }
+
+    }
+
+TEST(Imgproc_Mine, accuracy) {
```

Comment:
Please use some meaningful name for the test case and the file. Also I recommend to place it to the same file where the original function was tested.

Reply:
done.